pkgs/development/ocaml-modules/ppxlib/default.nix

```diff
@@ -2,7 +2,7 @@
 , version ?
   if lib.versionAtLeast ocaml.version "4.07"
   then if lib.versionAtLeast ocaml.version "4.08"
-  then "0.23.0" else "0.15.0" else "0.13.0"
+  then "0.24.0" else "0.15.0" else "0.13.0"
```

Comment:
@vbgl Where does this logic come from? From what I can see in the ppxlib changelog and in the opam files, recent ppxlib versions are compatible with OCaml >= 4.04.

Reply:
You can try by yourself and see what fails.

Something that comes to mind is the version of `sexplib0` that is packaged in `nixpkgs` for OCaml < 4.07